source/main/ipc/getHardwareWalletChannel.ts

```diff
@@ -85,22 +99,14 @@ class EventObserver {
                 AdaConnection,
               };
               this.getHardwareWalletConnectionChannel.send(
-                {
-                  disconnected: false,
-                  deviceType: 'ledger',
-                  deviceId: null,
-                  // Available only when Cardano APP opened
-                  deviceModel: deviceModel.id,
-                  // e.g. nanoS
-                  deviceName: deviceModel.productName,
-                  // e.g. Test Name
-                  path: device.path,
-                  product: device.product,
-                },
+                walletData,
                 this.mainWindow
               );
             } catch (e) {
```

Comment:
You could also rename this argument to `error` and use it without the `: e` bit in line 108 :)